src/helpers/serializer.ts

```diff
@@ -0,0 +1,73 @@
+import { CollectionQueryType, WhereArray, WhereItem, WhereType } from '..'
```

Comment:
At first glance this looks like a circular import. Maybe we should move these types into their own file.

Reply:
Yes, definitely, I will do that as well.